.github/workflows/publish-latest.yml

```diff
@@ -21,6 +25,13 @@ jobs:
       - name: Check if repo has unconsumed changesets
         id: has-changesets
         uses: ./.github/actions/has-changesets
+      # Note: if there are no changesets, that means that either
+      #         (1) "Version Packages" PR was just merged, or
```

Comment:
I like how you indent the comment, did you use any tool for that or manually indented?

Reply:
They are manually indented 😅